ipfs-cluster-service/main.go

```diff
@@ -379,11 +378,11 @@ func initConfig(force bool, generateSecret bool, envSecret string) {
 	cfg, err := ipfscluster.NewDefaultConfig()
 	checkErr("creating default configuration", err)
 
-	if len(envSecret) != 0 {
+	if envSecret != nil {
 		// read cluster secret from env variable
 		fmt.Println("Reading cluster secret from CLUSTER_SECRET environment variable.")
-		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(envSecret)
-	} else if !generateSecret {
+		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(*envSecret)
```

Comment:
I think this will break when envSecret = "" ? Will the Decode throw an error then?